bindings_ffi/src/mls.rs

```diff
@@ -6967,7 +6967,7 @@ mod tests {
         // Wait for alix_a to send out the consent on the sync group
         alix_a
             .worker()
-            .wait(SyncMetric::ConsentSent, 2)
```

Comment:
This undoes the unit test changes from #2201. The test technically still passes even with the legacy device sync payloads being marked as ERROR, but if we are adding them back then we need to wait for them in order for the test to succeed.